### PR TITLE
Add Greg and Sebastian as Editors to EdDSA Cryptosuites

### PR DESCRIPTION
**Pull request by msporny**
This PR adds Greg and Sebastian as Editors to the EdDSA Cryptosuites. The PR shouldn't be processed until after Sebastian's IE application has been processed. /cc @Wind4Greg @seabass-labrax

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-di-eddsa/pull/50.html" title="Last updated on Jul 13, 2023, 9:53 PM UTC (ecfae04)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-di-eddsa/50/61d9296...ecfae04.html" title="Last updated on Jul 13, 2023, 9:53 PM UTC (ecfae04)">Diff</a>

### CODE REVIEW

**Comment by iherman** [comment]
> This PR adds Greg and Sebastian as Editors to the EdDSA Cryptosuites. The PR shouldn't be processed until after Sebastian's IE application has been processed. /cc @Wind4Greg @seabass-labrax
>

I agree, but we have never received a formal application of @seabass-labrax (only an informal request to the chairs). Sebastian, you should fill [the application form](https://www.w3.org/groups/wg/vc/apply-as-invited-expert/).

File: index.html
Comment (seabass-labrax) [inline index.html]:
```suggestion
          company: "Invited Expert", w3cid: 140132
```

**Review by seabass-labrax — CHANGES_REQUESTED** [review]

File: index.html
Comment (msporny) [inline index.html]:
```suggestion
          company: "Invited Expert", w3cid: 140479
```

**Review by dmitrizagidulin — APPROVED** [review]

**Comment by brentzundel** [comment]
Minor concern here that at this point Sebastian has been accepted as an Invited Expert, but not yet joined the VCWG.

**Comment by msporny** [comment]
> Minor concern here that at this point Sebastian has been accepted as an Invited Expert, but not yet joined the VCWG.

Yes, he is reading through the IPR policy and WG Charter to make sure he knows what he's signing up for... we were going to wait to merge this until he had joined the WG. 

I guess at this point, we can just not accept PRs from him until he has joined the WG.

@dmitrizagidulin, we need to wait for 7 days before we can merge PRs given the WG's work mode:

We need to wait 7 days for comments before being able to merge PRs and then be clear about the disposition of the PR before merging, namely:

1. Was the change editorial, normative, and/or substantive?
2. Did it have multiple reviewers (it needs to have multiple reviewers, at least one being an Editor, and none of the reviewers should be the person that created the PR)?
3. Were there changes requested and were all substantial requested made/addressed?
4. If there were changes requested that were not made, are they being tracked by issue(s)?
5. Were there any objections (there shouldn't be any when you merge)?

So, the line for this PR (as long as no one makes changes/objects in the next two days) would be:

Editorial, multiple reviewers, no changes requested, no objections, merging.

**Pull request merged**
